index.jsx

```diff
@@ -481,7 +491,7 @@ class Controls extends Component {
         <div>
           <button
             ref={(node) => { this.minusNode = node; }}
-            className={btnClass}
+            className={`${btnClass} ${minusBtnClass}`}
```

Comment:
If `btnClass` or `minusBtnClass` is `undefined` then this will add `undefined` as a class. Instead of using string interpolation I would just add classnames a dependency and use it: https://www.npmjs.com/package/classnames

Reply:
I suggest to approach it a little bit differently. Your react plugin is unique in a way that it has no dependencies and I'd like to keep it this way. We can achieve the same result using `join(' ')`:
```jsx
className={[
  btnClass ? btnClass : '',
  minusBtnClass ? minusBtnClass : '',
].join(' ')}
```